packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -70,10 +74,19 @@ export const CameraAndMicrophoneDomainPermissions = (props: CameraAndMicrophoneD
   /* @conditional-compile-remove(call-readiness) */
   const locale = useLocale().strings;
 
+  /* @conditional-compile-remove(call-readiness) */
+  const isSafari = (): boolean => {
```

Comment:
you should extract this, and the function on lines 147 and 208 to just be 1 function

Reply:
Good call, fixed that